db/schema.rb

```diff
@@ -101,6 +107,9 @@
     t.index ["tenant_id"], name: "index_orders_on_tenant_id"
   end
 
+# Could not dump table "permissions" because of following StandardError
```

Comment:
@syncrou Is this a comment or some other issue?

Reply:
This is because the rails schema dump rake task doesn't know how to handle the postgres enum type.

Reply:
@syncrou There are some articles that mention to overcome this problem using
config.active_record.schema_format = :sql

And switch from db/schema.rb to db/structure.sql